### ISSUE TITLE
[Bug] [Code: ATWxAF] <your bug title>

### ISSUE THREAD

**Issue by gaige** [FlineDev/ReMafoX#43]
## Steps to Reproduce:

1. Renamed my project directory (in which the ReMafoX project file and my source exist
2. Double-clicked on the ReMafoX project

## Expected Behavior:

Open and allow me to work on the renamed project

## Current Behavior:

Error message and not able to work with the file

## Environment

ATWxAF

<details>
<summary>Show environment details</summary>

- App Version: 1.0.2 (Build 51)
- System Version: 12.6 (Build 21G115)
- System CPU: Intel(R) Xeon(R) W-3245 CPU @ 3.20GHz

</details>

**Comment by Jeehut** [comment]
Thank you for reporting this.

You're right, if you rename a project, you need to re-provide access to the project and the app does state that it no longer has access, but it doesn't help with fixing that and it should be improved. As you can see though, users who come across this again will see this issue linked:
<img width="1052" alt="Screen Shot 2022-10-15 at 22 20 44" src="https://user-images.githubusercontent.com/6942160/196006105-cdf9ef24-b382-49ea-af89-8b673413bee8.png">

So until I provide better explanations right within the app, here's how to fix it if you run into this:

1. Cancel the Add Translation workflow for now
2. Open the ReMafoX app, open the Projects Browser if needed (Cmd + P)
3. Delete your configured project from the list on the right by pressing the 🗑️ button
4. Then double-press the `.remafox` config file in your project to start the quick setup
5. Press the "Quick Config" button and confirm access to the folder by pressing "Allow"

That's it, now you can start the Add Translation workflow again and it should work. Just remember to select the right project if you have multiple set up on first usage of the window.

I hope this helps!

**Comment by Jeehut** [comment]
@gaige If I follow your reproduction steps and double-click the project in the Projects Browser after moving a project, I get this error, which is different than what you report in the title:

<img width="1053" alt="Screen Shot 2022-10-16 at 08 07 04" src="https://user-images.githubusercontent.com/6942160/196020872-5eafab5b-7dea-46e9-b63f-dec72852ddba.png">

As you can see, there's already a clear guide on what to do in this error message to fix it. To reproduce the error you reported you have to start the Add Translation window and select the project if not already selected. I will still make sure to add a similar (if not the same) message to this use case as well.

**Comment by gaige** [comment]
Agreed, my reproduction steps are not consistent with what I remember doing, which was clicking on the Project in the Project Browser.

**Comment by Jeehut** [comment]
As I described [here](https://github.com/FlineDev/RemafoX/issues/92#issuecomment-1504298698), I improved the handling of these situations, both with the wording of the error and also with an extra dialog. The fix will ship with `1.5.0` in a couple of days.

@gaige Thank you again for reporting this issue!
